projects/plugins/jetpack/modules/sso.php

```diff
@@ -83,6 +88,52 @@ private function __construct() {
 		}
 	}
 
+	/**
+	 * Render the invitation email message.
+	 */
+	public function render_invitation_email_message() {
+		printf( '<div class="notice inline notice-large notice-info">%1s</div>', esc_html__( 'The user will be automatically invited to WordPress.com.', 'jetpack' ) );
```

Comment:
It seems we can use [this function](https://developer.wordpress.org/reference/functions/wp_admin_notice/) btw.

Reply:
As far as I have researched, we can indeed use `wp_admin_notice` to render the notice message, the drawback is that `wp_admin_notice` renders it at the top of the page. I haven't found a configurable way of rendering it in a specific position.

Reply:
We have decided to mo the message up to keep the consistency and make it easier for the users, using the notice function 👍